samplefigure.py

```diff
@@ -23,3 +23,31 @@
 # sometimes elements get cut off on the edges.
 plt.tight_layout()
 plt.savefig('samplefigure.pdf')
+
+#Lets investigate the effect of noise on our Model "A" and it's derivative
+for index, value  in enumerate([1.0, 10, 100]):
```

Comment:
Good idea to generate all these files seperately, but we really should be using a function to do the decorations and plotting rather than copy/pasting all these blocks. I want to model best practice.

Reply:
I struggled to figure out how far to go when creating these functions, I've made 2 functions and while they aren't universal in their application, they do make the for-loop only 4 lines long

Reply:
I'm OK with it as it stands right now. We can add a refactor to the issue list.